panel/layout/modal.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar, Mapping
+
+import param
+
+from ..io.resources import CDN_DIST
+from ..models import Modal as BkModal
+from ..models.layout import ModalDialogEvent
+from .base import ListPanel
+
+if TYPE_CHECKING:
+    from bokeh.model import Model
+
+
+class Modal(ListPanel):
+    height = param.Integer(default=None, bounds=(0, None))
+
+    width = param.Integer(default=None, bounds=(0, None))
+
+    is_open = param.Boolean(default=False, readonly=True, doc="Whether the modal is open.")
```

Comment:
Love this PR.

I've experienced that for some use cases I would like the `open` and `close` methods. For some it was better if I could set `is_open=True` or `is_open=False`. Please consider the last feature :-)

Reply:
@MarcSkovMadsen I also missed these two methods in panel_modal. I ended up using `modal._show()`.